ui/hooks/src/use-pending-reflections.ts

```diff
@@ -0,0 +1,26 @@
+import { makeVar, useReactiveVar } from '@apollo/client';
+import type { ReflectEntryData } from '@akashaorg/typings/lib/ui';
+
+const pendingReflectionsVar = makeVar([]);
```

Comment:
This being global doesn't introduce weird bugs when adding reflections to different beams? I'm thinking about using an init function for this, wouldn't this work?

Reply:
or initialize this at the component level and pass it to the `usePendingReflections`

Reply:
ok so it should better to reset it on navigation rather than dangling around.
the only 'downside' will be in case if the publishing takes a bit longer and you navigate from and immediately back to it. but i guess this is quite an edge case